app/livechat/client/views/app/livechatInstallation.js

```diff
@@ -2,10 +2,10 @@ import { Template } from 'meteor/templating';
 import { settings } from '/app/settings';
 import s from 'underscore.string';
 
-Template.livechatInstallation.helpers({
-	script() {
-		const siteUrl = s.rtrim(settings.get('Site_Url'), '/');
+const siteUrl = s.rtrim(settings.get('Site_Url'), '/');
```

Comment:
please move this back to inside the helper (or to any other reactive place), because currently when the page loads the setting has no value yet, so the script ends up **without** this vital information